databricks/koalas/series.py

```diff
@@ -3250,28 +3265,43 @@ def idxmin(self, skipna=True):
 
         >>> s.idxmin()
         ('b', 'f')
+
+        If multiple values equal the minimum, the first row label with that
+        value is returned.
+
+        >>> s = ks.Series([1, 100, 1, 100, 1, 100], index=[10, 3, 5, 2, 1, 8])
+        >>> s
+        10      1
+        3     100
+        5       1
+        2     100
+        1       1
+        8     100
+        Name: 0, dtype: int64
+
+        >>> s.idxmin()
+        10
         """
         sdf = self._internal._sdf
         scol = self._scol
         index_scols = self._internal.index_scols
-        # asc_nulls_(list|first)is used via Py4J directly because
-        # it's not supported in Spark 2.3.
         if skipna:
-            sdf = sdf.orderBy(Column(scol._jc.asc_nulls_last()))
```

Comment:
@itholic the test failure against Spark 2.3.x is because you replaced this `Column(scol._jc.asc_nulls_last())` -> `scol.asc_nulls_last()`.

Reply:
> asc_nulls_(list|first)is used via Py4J directly because it's not supported in Spark 2.3.